src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -43,7 +43,7 @@ websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selec
 websocket.manual_send.menu                      = Manual Send WebSocket Message
 websocket.manual_send.menu.mnemonic				= w
 websocket.manual_send.popup                     = Resend WebSocket Message
```

Comment:
I didn't mean to use the same name as core (HTTP), but use the same for both (WebSocket) dialogues ("Resend WebSocket Message" and "Manual Send WebSocket Message").

That name works for me.